calico_cni/calico_cni.py

```diff
@@ -435,8 +462,8 @@ def _get_endpoint(self):
             _log.warning("No endpoint found matching ID %s", self.container_id)
             endpoint = None
         except MultipleEndpointsMatch:
-            _log.error("Multiple endpoints found matching ID %s", self.container_id)
-            sys.exit(1)
+            _log.warning("Multiple endpoints found matching ID %s", self.container_id)
```

Comment:
I'm not sure if this is the right behavior here.  It is definitely an error if multiple endpoints are returned for a single container_id.